helix-core/src/main/java/org/apache/helix/monitoring/mbeans/dynamicMBeans/DynamicMBeanProvider.java

```diff
@@ -51,6 +52,7 @@
 public abstract class DynamicMBeanProvider implements DynamicMBean, SensorNameProvider {
   protected final Logger _logger = LoggerFactory.getLogger(getClass());
   protected static final long DEFAULT_RESET_INTERVAL_MS = 60 * 60 * 1000; // Reset time every hour
+  protected static final String RESET_INTERVAL_SYSTEM_PROPERTY_KEY = "reservoir.length.ms";
```

Comment:
Sorry that I missed this name in the previous review. Can we have a more specific name such as "helix.monitor.slidingTimeWindow.ms"?

And please move it to SystemPropertyKeys.java with the comments.

Reply:
Done.